lib/presentation/ui/occurrences_time_widget.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+
+class OccurrencesTimeWidget extends StatelessWidget {
+
+  final DateTime _startTime;
+  final DateTime _endTime;
+  final DateTime _lastUpdated;
+
+  const OccurrencesTimeWidget(this._startTime, this._endTime, this._lastUpdated);
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      type: MaterialType.transparency,
+      child: new Container (
+                decoration: new BoxDecoration(
+                  color: Colors.grey,
+                  boxShadow: [new BoxShadow(
+                  color: Colors.black,
+                  blurRadius: 20.0,
+                  )],
+                ),
+                child:
+                    new Column(
+                      children: <Widget>[
+                        new Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: <Widget>[
+                            new Column(
+                              mainAxisSize: MainAxisSize.min,
+                              children: <Widget>[
+                                new Text(
+                                  "Início",
+                                  overflow: TextOverflow.clip,
+                                  style: DefaultTextStyle.of(context).style.apply(
+                                      fontSizeFactor: 0.5,
+                                      decoration: TextDecoration.none,
+                                      color: Colors.grey
+                                  ),
+                                ),
+                                new Text(
+                                  this._startTime != null ? this._startTime.hour.toString() + ":" + this._startTime.minute.toString() : "--",
```

Comment:
You are using this same line of code twice.

Can you extract a method for it?

Also, consider using Date Formatting as seen here:
https://stackoverflow.com/a/16126580/4499889

Reply:
Also, to make it more readable, consider using `( ` and `)` on your logic

So:

`this._startTime != null ? (this._startTime.hour.toString() + ":" + this._startTime.minute.toString()) : "--",`

Reply:
@Vanethos , please look at my updated PR